src/app/components/PageLayout/NetworkSelector.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { styled } from '@mui/material/styles'
+import { NetworkButton, MobileNetworkButton } from './NetworkButton'
+import { COLORS } from '../../../styles/theme/colors'
+import { Network } from '../../../types/network'
+import { Layer } from '../../../oasis-indexer/api'
+
+export const StyledBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  borderColor: COLORS.brandExtraDark,
+  borderStyle: 'solid',
+  borderWidth: '1px 1px 1px 0',
+  padding: `${theme.spacing(2)} ${theme.spacing(3)} ${theme.spacing(2)} ${theme.spacing(2)}`,
+  borderTopRightRadius: '9px',
+  borderBottomRightRadius: '9px',
+  boxShadow: `inset 0px 4px 4px rgba(34, 47, 63, 0.24)`,
+}))
+
+const getNetworkLabels = (t: TFunction): { [key in Network]: ReactNode } => ({
```

Comment:
We also have this at #408: https://github.com/oasisprotocol/explorer/blob/91d4fc66e7ed466dc88022f04bd5ba480da78457/src/types/network.ts#L15

It would be nice to use the same name, so that when both are merged, it's easier to unify the code.